docs/usage/custom-device-files.md

```diff
@@ -1,12 +1,23 @@
-# Testing custom devices config
+# Using custom devices configuration files
 
-Zwave-js has an iternal database where it stores its device configuration files. Should you wish to modify a device file, or if you need to test a custom config device file, you may do so by placing the file inside a folder in the **store directory** named `config` (By default it is: `<app-root-folder>/store/config`).
+Z-Wave JS has an internal database where it stores its device configuration files. Should you wish to modify a device file, or if you need to test a custom config device file, you may do so by creating a folder inside your store folder named `config` and placing the file inside that folder. (By default it is: `<app-root-folder>/store/config`).
 
-For additional information, see the zwave-js [docs](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=zwaveoptions):
+This directory does not get indexed and should be used sparingly, e.g. when custom files are absolutely necessary or for testing. This directory does not expect any special structure. It can be organized as you like.
```

Comment:
It doesn't expect a structure most of the time, but if you're going to test templated files, you'll have to mimic the structure required by the templates.